components/recurring-contributions/UpdatePaymentMethodPopUp.js

```diff
@@ -171,6 +173,8 @@ const UpdatePaymentMethodPopUp = ({
         first(paymentOptions.filter(option => option.id === contribution.paymentMethod.legacyId)),
       );
       setLoadingDefaultPaymentMethod(false);
+    } else if (paymentOptions && addedPaymentMethod) {
+      setSelectedPaymentMethod(first(paymentOptions.filter(option => option.id === addedPaymentMethod.legacyId)));
```

Comment:
I have a personal preference for using `find` like below, but both are valid

```suggestion
      setSelectedPaymentMethod(paymentOptions.find(option => option.id === addedPaymentMethod.legacyId));
```